### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by Morganov**

### CODE REVIEW

File: book/03-git-branching/sections/rebasing.asc [hunk 1]
```diff
@@ -201,8 +201,8 @@ So instead of the result we see in <<_merge_rebase_work>>, we would end up with
 .Rebase on top of force-pushed rebase work.
 image::images/perils-of-rebasing-5.png[Rebase on top of force-pushed rebase work.]
 
-This only works if C4 and C4' that your partner made are almost exactly the same patch.
-Otherwise the rebase won't be able to tell that it's a duplicate and will add another C4-like patch (which will probably fail to apply cleanly, since the changes would already be at least somewhat there).
+This only works if `C4` and `C4'` that your partner made are almost exactly the same patch.
+Otherwise the `rebase` won't be able to tell that it's a duplicate and will add another C4-like patch (which will probably fail to apply cleanly, since the changes would already be at least somewhat there).
```
Comment (ben) [inline book/03-git-branching/sections/rebasing.asc]: The word "rebase" here is just referring to the rebasing operation, so it shouldn't be monospace.

File: book/03-git-branching/sections/remote-branches.asc [hunk 2]
```diff
@@ -80,9 +80,9 @@ To https://github.com/schacon/simplegit
 ----
 
 This is a bit of a shortcut.
-Git automatically expands the `serverfix` branchname out to `refs/heads/serverfix:refs/heads/serverfix`, which means, ``Take my serverfix local branch and push it to update the remote's serverfix branch.''
+Git automatically expands the `serverfix` branchname out to `refs/heads/serverfix:refs/heads/serverfix`, which means, ``Take my `serverfix` local branch and push it to update the remote's `serverfix` branch''.
```
Comment (ben) [inline book/03-git-branching/sections/remote-branches.asc]: Apparently there is [some debate](https://style.mla.org/punctuation-and-quotation-marks/) as to whether periods should go inside quotation marks, in this book we've settled on the "inside rule."

File: book/03-git-branching/sections/remote-branches.asc [hunk 3]
```diff
@@ -80,9 +80,9 @@ To https://github.com/schacon/simplegit
...
 We'll go over the `refs/heads/` part in detail in <<ch10-git-internals#ch10-git-internals>>, but you can generally leave it off.
-You can also do `git push origin serverfix:serverfix`, which does the same thing -- it says, ``Take my serverfix and make it the remote's serverfix.''
+You can also do `git push origin serverfix:serverfix`, which does the same thing -- it says, ``Take my serverfix and make it the remote's serverfix''.
```
Comment (ben) [inline book/03-git-branching/sections/remote-branches.asc]: Again, periods inside quotation marks please.

File: book/03-git-branching/sections/rebasing.asc (hunk 1 above)
Comment (Morganov) [reply book/03-git-branching/sections/rebasing.asc]: My bad, fixed.

File: book/03-git-branching/sections/remote-branches.asc (hunk 2 above)
Comment (Morganov) [reply book/03-git-branching/sections/remote-branches.asc]: Ok. Fixed.

**Comment by ben** [comment]
✨ Thanks!

**Pull request merged**